content/en/docs/architecture/retester.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Retester"
+description: An overview of the retester implementation.
+---
+## What Is Retester?
+Retester is a tool that evaluate which pull requests should be retested, and then selected pull requests retest 
+by [@openshift-ci-robot](https://github.com/openshift-ci-robot) 's commenting `/retest-required` on the pull requests.
+
+## How Do We Set up Retester?
+Retester completely reuses [`tide`](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md) 
+configuration, whatever onboards to Tide, gets onboarded to retester too.
+Retester is aware of Prow and the concept of optional and required jobs. It only triggers retests on PRs 
+where at least one required Prow job is failing.
+
+We need to enable repository on the retester's deployment and remove it from the old retester.
+
+{{< highlight yaml >}}
+  spec:
+    containers:
+    - image: retester:latest
+      name: retester
+      command:
+      - retester
+      args:
+      - --config-path=/etc/config/config.yaml
+      - --supplemental-prow-config-dir=/etc/config
+      - --dry-run=false
+      - --job-config-path=/etc/job-config
+      - --cache-file=/cache/backoff
+      - --enable-on-repo=openshift/ci-tools       #Add this line in new retester's deployment
```

Comment:
Does it support regex, e.g., `--enable-on-repo=openshift/*`? Or `--enable-on-org`?

Reply:
No it's only `--enable-on-repo=<org>/<repo>` but you can use this command multiple times.

Reply:
Could you please create a card for this? Because we are going to enable it on the openshift org.

Reply:
created: [DPTP-2853](https://issues.redhat.com/browse/DPTP-2853)